.github/workflows/pytest.yml

```diff
@@ -121,7 +121,7 @@ jobs:
 
           if [ "$PYTORCH" == "nightly" ]; then
             extra_index_url=https://download.pytorch.org/whl/nightly/cpu
-            pip install --pre torch torchtext torchvision torchaudio --index-url $extra_index_url
+            pip install --pre torch==2.1.0.dev20230508+cpu torchtext torchvision torchaudio --index-url $extra_index_url
```

Comment:
Can we create a GH issue to remember to unpin torch nightly

Reply:
As it's a new day, I'll give torch nightly another try.

Reply:
If it fails, I'll add the pin back and file an issue.

Reply:
Still failing. Added pin back and filed an issue.